clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/AbandonAndDecrementGoogleServerGroupAtomicOperation.groovy

```diff
@@ -67,12 +67,23 @@ class AbandonAndDecrementGoogleServerGroupAtomicOperation implements AtomicOpera
     def region = description.region
     def serverGroupName = description.serverGroupName
     def serverGroup = GCEUtil.queryServerGroup(googleClusterProvider, accountName, region, serverGroupName)
+    def isRegional = serverGroup.regional
+    // Will return null if this is a zonal server group.
```

Comment:
It seems this line has been c/p'ed a few times. Did you mean to say "Will return null if this is a **regional** server group"?